subsys/partition_manager/Kconfig

```diff
@@ -82,6 +82,12 @@ partition-size=0x1000
 rsource "Kconfig.template.partition_size"
 endif
 
+if EMDS
```

Comment:
Im not quite sure, but should there be a combo setting for EMDS && SETTINGS here?

Reply:
I am not sure what you mean with "combo setting". But I really want the EMDS to be totally independent on SETTINGS. So it can be used with or without settings.